script/worker

```diff
@@ -120,7 +120,6 @@ sub read_worker_config($){
             }
         }
     }
-    $sets->{'BACKEND'} ||= "qemu";
```

Comment:
If someone does not provide BACKEND in workers.ini, I think this will trigger uninitialized variable during register_worker:338